pkgs/applications/editors/jetbrains/bin/linux.nix

```
@@ -116,7 +116,7 @@ with stdenv; lib.makeOverridable mkDerivation (rec {
       --set-default LOCALE_ARCHIVE "${glibcLocales}/lib/locale/locale-archive" \
       --set-default ${hiName}_VM_OPTIONS ${vmoptsFile}
 
-    ln -s "$out/$pname/bin/${loName}.sh" $out/bin/$pname
+    ln -s "$out/$pname/bin/${loName}" $out/bin/$pname
```

This breaks for IDEs that have not yet hit 2024.2, such as RustRover.

```suggestion
    launcher="$out/$pname/bin/${loName}"
    if [ ! -e "$launcher" ]; then
      launcher="$out/$pname/bin/${loName}.sh"
    fi
    ln -s "$launcher" $out/bin/$pname
```

---

Ah. Thanks for catching and fixing!